.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: linting
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1.1
+        with:
+          version: 3.*
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
```

Comment:
I'd recommend using `3.x` here to stay up to date unless there's a known particular compatibility issue. 3.7 is only due another couple of years of security patches.

Reply:
ok